modules/install-openshift-common-terms.adoc

```diff
@@ -6,16 +6,16 @@
 [id="install-openshift-common-terms_{context}"]
 = Glossary of common terms for {product-title} installing
 
-This glossary defines common terms that are used in the installation content. These terms help you understand installation effectively.
+The glossary defines common terms that relate to the installation content. Read the following list of terms to better understand the installation process.
 
 {ai-full}::
-An installer hosted at link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index[console.redhat.com] that provides a web user interface or a RESTful API for creating a cluster configuration. The {ai-full} generates a discovery image. Cluster machines boot with the discovery image, which installs {op-system} and an agent. Together, the {ai-full} and agent provide pre-installation validation and installation for the cluster.
+An installer hosted at link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index[console.redhat.com] that provides a web-based user interface or a RESTful API for creating a cluster configuration. The {ai-full} generates a discovery image. Cluster machines boot with the discovery image, which installs {op-system} and an agent. Together, the {ai-full} and agent provide pre-installation validation and installation for the cluster.
 
-Agent-based installer::
-An installer similar to the {ai-full}, but you must download the link:https://console.redhat.com/openshift/install/metal/agent-based[agent-based installer] first. The agent-based installer is ideal for air-gapped/restricted networks.
+Agent-based Installer::
+An installer similar to the {ai-full}, but you must download the link:https://console.redhat.com/openshift/install/metal/agent-based[Agent-based Installer] first. The Agent-based Installer is ideal for disconnected environments and restricted networks.
```

Comment:
```suggestion
An installer similar to the {ai-full}, but you must download the link:https://console.redhat.com/openshift/install/metal/agent-based[Agent-based Installer] first. The Agent-based Installer is ideal for disconnected environments.
```
Per https://redhat-documentation.github.io/supplementary-style-guide/#disconnected, these terms are redundant and "disconnected" is the preferred term.